install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -479,7 +479,7 @@ data:
   node-port-mode: {{ .Values.nodePort.mode | quote }}
 {{- end }}
 {{- if hasKey .Values.nodePort "algorithm" }}
-  node-port-algorithm: {{ .Values.nodePort.algorithm | quote }}
+  bpf-lb-algorithm: {{ .Values.nodePort.algorithm | quote }}
 {{- end }}
```

Comment:
We're changing a bunch of other Helm options this release, so if we were to also rename the other acceleration/mode options in Helm then this wouldn't be a bad time to do so.

Reply:
Thanks, do you have some pointers to commits from our side & how we retain backwards compat?

Reply:
The current approach of the Helm changes replaces every option with an equivalent, differently-named option which the user must specify (using the helm options table in the upgrade guide: https://docs.cilium.io/en/latest/operations/upgrade/#helm-options)

Reply:
Nice, I'll rename the LB ones then, but in new PR.